commcare_connect/opportunity/helpers.py

```diff
@@ -151,3 +172,92 @@ def get_payment_report_data(opportunity: Opportunity):
             PaymentReportData(payment_unit.name, completed_work_count, user_payment_accrued, nm_payment_accrued)
         )
     return data, total_user_payment_accrued, total_nm_payment_accrued
+
+
+def get_opportunity_list_data(organization, program_manager=False):
```

Comment:
is this performant? just making sure we aren't building something that will get too slow very quickly

Reply:
Yes, it's likely this is not performant enough. We are setting aside time to look into this and other places with these kind of queries